products/identity/Products/Identity Dashboard/Identity_Dashboard/Managing Teams and Users.md

```diff
@@ -0,0 +1,130 @@
+* The request should contain the following information:
+  * The environment of the team.
+  * The `name` of the team.
+  * Any details that need to be updated for the team, which may include:
+    * `name`: the human-readable name for the team.
+    * `description`: a brief description of the team, including the systems, processes, or verticals the team supports.
+    * Admin details, including:
+      * `logingov_uuid`: The UUID associated with the admin's Login.gov account.
+      * `email`: The email address associated with the admin's Login.gov account.
+      * `first_name`: The first name for the admin.
+      * `last_name`: The last name for the admin.
+* Prior to executing the request, the OCTO-Identity team will verify the requestor is an admin for the specified team; requests from non-admins will be denied.
+* The OCTO-Identity team will then update the specified team using the details provided.
+
+### Removing a Team
+
+**Please note that any request to remove a team must be sent from a user with an `admin` role for that team.**
+
+**Removing a team is a dangerous operation as it will result in the deletion of any client configs associated with that team.**
+
+Follow these steps to remove a team:
+
+* Submit a request to the OCTO-Identity team via the [#vsp-identity Slack channel](https://dsva.slack.com/archives/CSFV4QTKN) to remove a team.
+* The request should contain the following information:
+  * The environment of the team.
+  * The `name` of the team.
+* Prior to executing the request, the OCTO-Identity team will verify the requestor is an admin for the specified team; requests from non-admins will be denied.
+* If applicable, the OCTO-Identity team will inform the requestor of any client configurations that will be impacted by the removal of the team prior to executing the request.
+* The OCTO-Identity team will then remove the specified team.
+
+## Team Users
+
+Teams contain users who have roles for managing teams and client application configurations within ID. These roles provide granular access control for team members depending on their role.
+
+For instance, team leads will likely need the ability to administer their teams (updating team information, adding new team users, etc.), but may not require full access to update the 
+client application configurations for which their teams are responsible. Conversely, engineers will almost certainly need access to administer client application configurations, but are unlikely to require permissions for administering teams.
+
+With respect to teams, users with an `admin` role can create new teams and manage the users that belong to those teams along with their roles. Users with a `member` role are restricted to viewing the existing teams and their associated users.
+
+With respect to client application configurations, users with an `admin` role can create and manage client configurations, whereas users with a `member` role can only view their teams' integrations.
```

Comment:
🗣️ I think this could be more clear... specifically, when I say _"...can only view their..."_, it seems to imply there is something to view (like an interface).

Reply:
The non-self service equivalent would be to request a copy of their team's information or a specific client application configuration. However, I don't have instructions for requesting information in the document currently. I can add some, but I wonder if that would represent a security concern (i.e. how do we deliver the requested information to the user securely?)

Maybe @joeniquette has thoughts.